src/transformers/models/dpt/image_processing_dpt.py

```diff
@@ -136,7 +136,7 @@ def __init__(
         self,
         do_resize: bool = True,
         size: Dict[str, int] = None,
-        resample: PILImageResampling = PILImageResampling.BILINEAR,
+        resample: PILImageResampling = PILImageResampling.BICUBIC,
```

Comment:
We can't just change defaults like this. If this is needed for a new model, then they can set it in their config.

Reply:
Ok, but I would see this more as a bug fix. Happy to revert if required

Reply:
Was the original addition not aligned with the original implementation?

Reply:
The original implementation uses cv2.INTER_CUBIC, which would be BICUBIC in Pillow: https://github.com/isl-org/MiDaS/blob/bdc4ed64c095e026dc0a2f17cabb14d58263decb/midas/model_loader.py#L221

Reply:
This has happened quite a few times now and should really not happen at all. Is there a reason why this isn't being caught when the models are being implemented? Is there a test or an extra step we should incorporate to avoid this?

Reply:
DPT was contributed at a time where pixel values weren't asserted against the original preprocessing pipelines. This is something that I've been doing since about a year now, e.g. here: https://github.com/huggingface/transformers/blob/main/src/transformers/models/dinov2/convert_dinov2_to_hf.py#L220.

Reply:
In this case, can you create a github issue for models old than a year with a checklist to verify the processing constants (and work through them)?

Reply:
Before merging, @NielsRogge - can you create and track this issue please? Otherwise it will be lost and not happen